app-config.yaml

```diff
@@ -42,16 +42,32 @@ auth:
 
 catalog:
```

Comment:
This should be marked readonly now right?! https://github.com/backstage/backstage/pull/5034

Reply:
Oh maybe that isn't released...

Reply:
Also I think we still might want to throw our own errors for nice error messages in the frontend for the demo site either way?

Reply:
@benjdlambert Do you mean we should include `catalog-import` for demo purposes but throw errors if you try to create something? The catalog backend does [reject post/delete](https://github.com/backstage/demo/blob/master/packages/backend/src/plugins/catalog.ts#L32), I could test how that manifests on the frontend. I had removed registration for now assuming it wouldn't be viable on the demo.

Reply:
I don't think we have the catalog plugin integrated right? I'd prefer to keep it that way as it's not going to do anything anyway. I meant keeping the post/delete rejection too so people can't POST requests to register anything but with nice error messages. @freben is talking about the readonly mode but I'm not sure if the error messages are nice for demo purposes from that functionality so we'd probably want to keep the overrides in the router anyway.